src/routes/auth.js

```diff
@@ -0,0 +1,73 @@
+import express from 'express';
+import UsersController from '../controllers/signup.controller';
+import signUpSchema from '../middlewares/signUpValidation';
+
+const router = express.Router();
+
+router.post('/api/auth/signup', signUpSchema, UsersController.signUp); // API route for user to signup
```

Comment:
To avoid repeating yourself, you can add `/api/auth` in the `app.use` context in `src/index.js`.